tests/Infrastructure/League/Tactician/Middleware/CommandFailedWrapperTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+declare(strict_types=1);
+
+namespace MyOnlineStore\CommandBus\Tests\Infrastructure\League\Tactician\Middleware;
+
+use MyOnlineStore\CommandBus\Domain\Exception\CommandFailedException;
+use MyOnlineStore\CommandBus\Infrastructure\League\Tactician\Middleware\CommandFailedWrapper;
+use PHPUnit\Framework\TestCase;
+
+final class CommandFailedWrapperTest extends TestCase
+{
+    /**
+     * @var CommandFailedWrapper
+     */
+    private $middleware;
+
+    protected function setUp()
+    {
+        $this->middleware = new CommandFailedWrapper();
+    }
+
+    public function testWrapsAnyExceptionInCommandFailedException()
```

Comment:
Perhaps you could also add a test that verifies the path where no exception is thrown